app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+
+  after_create :update_user_posts_counter
+
+  private
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly note that a part of the requirement states that the `Post` model has a method that returns the 5 most recent comments for a given post. Please implement this.

Reply:
@Whoistolu for the review